src/libstd/sys/common/unwind/mod.rs

```diff
@@ -172,7 +172,7 @@ pub fn panicking() -> bool {
 #[inline(never)]
 #[no_mangle]
 #[allow(private_no_mangle_fns)]
```

Comment:
Possibly - not sure how deeply that lint looks. This function is now public within its module, but not public outside of the crate.